src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -220,9 +223,9 @@
                    (let [ws-name (get-workspace-name-string ws)
                          ws-href (let [x (:workspace ws)] (str (:namespace x) ":" (:name x)))
                          disabled? (= (:accessLevel ws) "NO ACCESS")]
-                     [{:name ws-name :href ws-href :status (:status ws) :disabled? disabled?}
-                      {:name ws-name :href ws-href :status (:status ws) :disabled? disabled?
-                       :protected? (get-in ws [:workspace :realm])}
+                     [{:name ws-name :href ws-href :status (:status ws) :disabled? disabled? :protected? (get-in ws [:workspace :isProtected])}
```

Comment:
I pulled this into the `let` and also pulled in some code duplication that I noticed in the first two cells.